app/views/my_modules/protocols.html.erb

```diff
@@ -39,7 +39,7 @@
     </a>
     </div>
     <div class="collapse" id="assigned-items-container">
-      ...
+      <%= render partial: "my_modules/inventories/inventories_list" %>
```

Comment:
In code we mostly use `repositories`. Can you rename partial.

Reply:
updated